# Define the goal
geometry_msgs/Pose[] poses_list
bool include_takeoff
bool include_land
--- 
# Define the result
string output
---
# Define a feedback message
float32 percent_complete